subsys/fs/shell.c

```diff
@@ -48,13 +48,9 @@ static struct fs_mount_t fatfs_mnt = {
 #error "No disk device defined, is your board supported?"
 #endif
 
-FS_LITTLEFS_DECLARE_CUSTOM_CONFIG(
-	lfs_data,
-	CONFIG_SDHC_BUFFER_ALIGNMENT,
-	SDMMC_DEFAULT_BLOCK_SIZE,
-	SDMMC_DEFAULT_BLOCK_SIZE,
-	SDMMC_DEFAULT_BLOCK_SIZE,
-	2 * SDMMC_DEFAULT_BLOCK_SIZE);
+FS_LITTLEFS_DECLARE_CUSTOM_CONFIG(lfs_data, CONFIG_SDHC_BUFFER_ALIGNMENT, SDMMC_DEFAULT_BLOCK_SIZE,
+				  SDMMC_DEFAULT_BLOCK_SIZE, SDMMC_DEFAULT_BLOCK_SIZE,
+				  2 * SDMMC_DEFAULT_BLOCK_SIZE);
```

Comment:
I would leave it as it was. There is not reason to cramp all these together and the definition was easier to understand and to match with `FS_LITTLEFS_DECLARE_CUSTOM_CONFIG` definition.

Reply:
Again, I only ran clang-format. I didn't edit those lines manually.

Reply:
Just leave all the other changes and not apply these. You can do mixed reset of your branch and do git add --patch and select lines you want in.